### PR TITLE
Add user table 

### PR DESCRIPTION
**Pull request by amitch23**
## Description

Adds a user model and migrations with 2 fields returned from a successful login to login.gov: `login_gov_uuid` and `login_gov_email`. Enforce that the logingov UUID is unique in the table to prevent multiple user creation with the same UUID at the DB level.

## Verification Steps

* [x] All tests pass.
* [x] The requirements listed in [Querying the Database Safely](https://github.com/transcom/mymove/blob/master/docs/backend.md#querying-the-database-safely) have been satisfied.
* [x] There are no [aXe](https://www.deque.com/products/aXe/) warnings for UI. n/a
* [x] This works in IE. n/a

## References

* [Pivotal story]https://www.pivotaltracker.com/story/show/155131585

### CODE REVIEW

File: migrations/20180227224236_create_users.up.fizz [hunk 1]
```diff
@@ -0,0 +1,7 @@
+create_table("users", func(t) {
+	t.Column("id", "uuid", {"primary": true})
+	t.Column("login_gov_uuid", "uuid", {})
+	t.Column("login_gov_email", "text", {})
+})
+
+raw("ALTER TABLE users ADD CONSTRAINT constraint_name UNIQUE (login_gov_uuid);")
```
Comment (breanneboland) [inline migrations/20180227224236_create_users.up.fizz]: So weird that Pop doesn't have a `{"unique": true}` option.

Comment (amitch23) [reply migrations/20180227224236_create_users.up.fizz]: I know! I tried and it didn't work. Fizz has an option for indices, but it doesn't seem like for other column fields.

Comment (breanneboland) [reply migrations/20180227224236_create_users.up.fizz]: The little omissions here and there in Pop are really strange and interesting. Or they will be until they keep me from doing something I need to. In the meantime, it was really useful to see how you worked around it.

Comment (macrael) [reply migrations/20180227224236_create_users.up.fizz]: Unique true would be a great opportunity for a PR

**Pull request merged**